Documentation/Contributing/development-flow.md

```diff
@@ -243,6 +248,65 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
+The various linters can selectively be run locally as follows.
+
+### Linting go code
+
+Linting of go code can be performed  in multiple ways:
+
+Firstly, one can use the standard `go vet` tool included with the golang installation.
+This is automatically performed by `make build` but rook provides separate make targets for
+developers' convenience:
+
+* `make vet`
+* `make go.vet`
+
+These are synonymous and essentially run `go vet` on rook's codebase.
+
+Secondly, the equally standard [golangci-lint tool](https://github.com/golangci/golangci-lint)  can be used:
+Run this from the rook checkout base directory:
+
+```console
+$ golangci-lint run
+```
+
+This requires `golangci-lint`to be installed locally and available in the PATH.
```

Comment:
Seems better to me to make a golangci-lint target than to spend time and breath telling users how to manually lint locally.

Reply:
@BlaineEXE  wrote:

> Seems better to me to make a golangci-lint target than to spend time and breath telling users how to manually lint locally.

I agree, and I wanted to do this anyway. I have done it locally and will push the extra commit soon.

Reply:
done.